core/templates/dev/head/exploration_editor/EditorServices.js

```diff
@@ -1839,3 +1890,354 @@ oppia.factory('explorationWarningsService', [
     };
   }
 ]);
+
+oppia.factory('humanReadableLostChangesService', function() {
```

Comment:
I would call this 'lostChangesService' and then the function in it 'makeHumanReadable()' to save on duplication.